portality/models/article.py

```diff
@@ -271,6 +292,9 @@ def _generate_index(self):
         issns += cbib.get_identifiers(cbib.P_ISSN)
         issns += cbib.get_identifiers(cbib.E_ISSN)
 
+        # get the issn from the journal bibjson
+        issns.append(cbib.journal_issn)
```

Comment:
There are now 3 ISSNs in each article record. PISSN and EISSN have been recorded in ```Article.bibjson()``` which written on the article's creation. These are the article's own ISSNs.

My new work has added ```journal_issn``` which is a copy of the preferred ISSN from the journal record, and added to the journal section of the article's bibjson. So, this is an additional way of identifying the article, via a link to its parent Journal.

My hope by appending it to the Article's index of ISSNs is to make it more likely to be findable by a Journal which is looking for its articles. I'm not completely convinced I've done enough for this to be the case.